scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/BucketShardId.java

```diff
@@ -47,8 +47,8 @@ public static BucketShardId of(int bucketId, int shardId) {
     return new AutoValue_BucketShardId(bucketId, shardId);
   }
 
-  public static BucketShardId ofNullKey(int shardId) {
-    return new AutoValue_BucketShardId(NULL_KEYS_BUCKET_ID, shardId);
```

Comment:
Does this mean we used to create `shardId` null key files instead just 1 for the entire sink?

Reply:
yep, we used to.